app/_src/gateway/plugin-development/quickstart.md

```diff
@@ -0,0 +1,478 @@
+---
+title: Quick Start
+book: plugin_dev
+chapter: 2
+---
+
+The following instructions are written to help you quickly build, test, and run your first 
+{{site.base_gateway}} custom plugin with [Lua](http://www.lua.org/). The remaining pages in 
+this section will provide details on other advanced topics related to 
+plugin development and best practices.
+
+{:.note}
+> **Note:** The following prerequisites are required to complete this guide
+>   * A `docker` CLI and associated Docker engine
+>   * `git` 
+>   * `curl`
```

Comment:
Just to make these more clear/consistent with how we present prereqs in other docs, I'd make a specific H2 prereqs section with these bullet points.

Reply:
Updated the pre-reqs in the latest